pypeit/arcimage.py

```diff
@@ -31,20 +30,16 @@ class ArcImage(processimages.ProcessImages, masterframe.MasterFrame):
         master_key (:obj:`str`, optional):
             The string identifier for the instrument configuration.  See
             :class:`pypeit.masterframe.MasterFrame`.
+        master_dir (str, optional): Path to master frames
```

Comment:
I generally follow:
```
variable_name (type):
    Description
```
I.e., indenting the description below the variable name.  (I used to start the description on the same line like this, but changed because it made it easier for me to format things... but that will depend on your editor.)

Reply:
got it